index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="UTF-8">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <nav class="menu">
+            <div class="logo"><h2>My Logo</h2></div>
+
+            <ul>
+                <li><a href="#" class="active" onclick="markMenuItem(this)">Portfolio</a></li>
+                <li><a href="#" onclick="markMenuItem(this)">About</a></li>
+                <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+            </ul>
+
+            <div id="menuBtn" class="active">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+
```

Comment:
I think your container of the section should be a `header` tag that will contain the logo and the navigation as this will effectively represent content meaning. i.e
````
<header>
<a href="#" >My logo</a>
<nav class="menu">
 <ul>
                <li><a href="#" class="active" onclick="markMenuItem(this)">Portfolio</a></li>
                <li><a href="#" onclick="markMenuItem(this)">About</a></li>
                <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
            </ul>
</nav>
</header>
````